<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>DNA sequencing QC metrics — target_dna_qc • Target Osteosarcoma Data</title>

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
<!-- Bootstrap -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/united/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script>

<!-- sticky kit -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



<meta property="og:title" content="DNA sequencing QC metrics — target_dna_qc" />

<meta property="og:description" content="DNA sequencing QC metrics" />
<meta name="twitter:card" content="summary" />



<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->


<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-93043521-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-93043521-1');
</script>

  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">Target Osteosarcoma Data</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.7</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/lcc_variants.html">Analysis of variants for LCC genes</a>
    </li>
  </ul>
</li>
      </ul>
      
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/seandavi/TargetOsteoAnalysis/">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      
      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>DNA sequencing QC metrics</h1>
    
    <div class="hidden name"><code>target_dna_qc.Rd</code></div>
    </div>

    <div class="ref-description">
    
    <p>DNA sequencing QC metrics</p>
    
    </div>

    <pre class="usage"><span class='fu'>target_dna_qc</span>()</pre>
        

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='no'>x</span> <span class='kw'>=</span> <span class='fu'>target_dna_qc</span>()</div><div class='output co'>#&gt; <span class='message'>Parsed with column specification:</span>
#&gt; <span class='message'>cols(</span>
#&gt; <span class='message'>  .default = col_double(),</span>
#&gt; <span class='message'>  bamFile = <span style='color: #BB0000;'>col_character()</span><span></span>
#&gt; <span class='message'>)</span></div><div class='output co'>#&gt; <span class='message'>See spec(...) for full column specifications.</span></div><div class='output co'>#&gt; <span class='message'>Parsed with column specification:</span>
#&gt; <span class='message'>cols(</span>
#&gt; <span class='message'>  .default = col_double(),</span>
#&gt; <span class='message'>  bamFile = </span><span style='color: #BB0000;'>col_character()</span><span></span>
#&gt; <span class='message'>)</span></div><div class='output co'>#&gt; <span class='message'>See spec(...) for full column specifications.</span></div><div class='output co'>#&gt; <span class='message'>Parsed with column specification:</span>
#&gt; <span class='message'>cols(</span>
#&gt; <span class='message'>  .default = col_double(),</span>
#&gt; <span class='message'>  bamFile = </span><span style='color: #BB0000;'>col_character()</span><span></span>
#&gt; <span class='message'>)</span></div><div class='output co'>#&gt; <span class='message'>See spec(...) for full column specifications.</span></div><div class='output co'>#&gt; <span class='message'>Parsed with column specification:</span>
#&gt; <span class='message'>cols(</span>
#&gt; <span class='message'>  .default = col_double(),</span>
#&gt; <span class='message'>  bamFile = </span><span style='color: #BB0000;'>col_character()</span><span></span>
#&gt; <span class='message'>)</span></div><div class='output co'>#&gt; <span class='message'>See spec(...) for full column specifications.</span></div><div class='input'><span class='fu'><a href='https://www.rdocumentation.org/packages/utils/topics/head'>head</a></span>(<span class='no'>x</span>)</div><div class='output co'>#&gt; </span><span style='color: #949494;'># A tibble: 6 x 46</span><span>
#&gt;   Sample Type  dataset bamFile Reads_generated Reads_mapped UNMAPPED_READS
#&gt;   </span><span style='color: #949494;font-style: italic;'>&lt;chr&gt;</span><span>  </span><span style='color: #949494;font-style: italic;'>&lt;chr&gt;</span><span> </span><span style='color: #949494;font-style: italic;'>&lt;chr&gt;</span><span>   </span><span style='color: #949494;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span>        </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span>          </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span>
#&gt; </span><span style='color: #BCBCBC;'>1</span><span> 0A4HLD Norm… discov… 0A4HLD…       283</span><span style='text-decoration: underline;'>058</span><span>872    272</span><span style='text-decoration: underline;'>586</span><span>395       10</span><span style='text-decoration: underline;'>472</span><span>477
#&gt; </span><span style='color: #BCBCBC;'>2</span><span> 0A4HLD Tumor discov… 0A4HLD…       170</span><span style='text-decoration: underline;'>825</span><span>466    166</span><span style='text-decoration: underline;'>919</span><span>706        3</span><span style='text-decoration: underline;'>905</span><span>760
#&gt; </span><span style='color: #BCBCBC;'>3</span><span> 0A4HMC Norm… discov… 0A4HMC…      </span><span style='text-decoration: underline;'>1</span><span>972</span><span style='text-decoration: underline;'>824</span><span>516   </span><span style='text-decoration: underline;'>1</span><span>936</span><span style='text-decoration: underline;'>120</span><span>262       36</span><span style='text-decoration: underline;'>704</span><span>254
#&gt; </span><span style='color: #BCBCBC;'>4</span><span> 0A4HMC Tumor discov… 0A4HMC…      </span><span style='text-decoration: underline;'>2</span><span>018</span><span style='text-decoration: underline;'>527</span><span>364   </span><span style='text-decoration: underline;'>1</span><span>987</span><span style='text-decoration: underline;'>585</span><span>279       30</span><span style='text-decoration: underline;'>942</span><span>085
#&gt; </span><span style='color: #BCBCBC;'>5</span><span> 0A4HX8 Norm… discov… 0A4HX8…       345</span><span style='text-decoration: underline;'>439</span><span>442    315</span><span style='text-decoration: underline;'>164</span><span>331       30</span><span style='text-decoration: underline;'>275</span><span>111
#&gt; </span><span style='color: #BCBCBC;'>6</span><span> 0A4HX8 Tumor discov… 0A4HX8…       374</span><span style='text-decoration: underline;'>712</span><span>362    361</span><span style='text-decoration: underline;'>488</span><span>204       13</span><span style='text-decoration: underline;'>224</span><span>158
#&gt; </span><span style='color: #949494;'># … with 39 more variables: Percentage_mapped </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, Reads_duplicated </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   Percentage_duplication </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, MEDIAN_INSERT_SIZE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   MAX_INSERT_SIZE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, STANDARD_DEVIATION </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, MEAN_INSERT_SIZE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   GENOME_TERRITORY </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_MEAN_COVERAGE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_SD_COVERAGE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   wgs_MEDIAN_COVERAGE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_PCT_EXC_UNPAIRED </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   wgs_PCT_EXC_CAPPED </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_PCT_EXC_TOTAL </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_PCT_5X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   wgs_PCT_10X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_PCT_20X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, wgs_PCT_50X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   wgs_PCT_100X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_ON_BAIT </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_NEAR_BAIT </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   MEAN_READ_LENGTH </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, BAIT_TERRITORY </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, TARGET_TERRITORY </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   MEAN_BAIT_COVERAGE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, MEAN_TARGET_COVERAGE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   PCT_TARGET_BASES_2X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_TARGET_BASES_20X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   PCT_TARGET_BASES_50X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_TARGET_BASES_100X </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   PCT_USABLE_BASES_ON_BAIT </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_USABLE_BASES_ON_TARGET </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   FOLD_ENRICHMENT </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, HS_LIBRARY_SIZE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, qMean </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>,
#&gt; #   PCT_EXC_DUPE </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_EXC_BASEQ </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, PCT_EXC_OVERLAP </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #949494;'>, `Average
#&gt; #   Insert Size` </span><span style='color: #949494;font-style: italic;'>&lt;dbl&gt;</span><span></div><div class='input'><span class='fu'><a href='https://www.rdocumentation.org/packages/base/topics/colnames'>colnames</a></span>(<span class='no'>x</span>)</div><div class='output co'>#&gt;  [1] "Sample"                     "Type"                      
#&gt;  [3] "dataset"                    "bamFile"                   
#&gt;  [5] "Reads_generated"            "Reads_mapped"              
#&gt;  [7] "UNMAPPED_READS"             "Percentage_mapped"         
#&gt;  [9] "Reads_duplicated"           "Percentage_duplication"    
#&gt; [11] "MEDIAN_INSERT_SIZE"         "MAX_INSERT_SIZE"           
#&gt; [13] "STANDARD_DEVIATION"         "MEAN_INSERT_SIZE"          
#&gt; [15] "GENOME_TERRITORY"           "wgs_MEAN_COVERAGE"         
#&gt; [17] "wgs_SD_COVERAGE"            "wgs_MEDIAN_COVERAGE"       
#&gt; [19] "wgs_PCT_EXC_UNPAIRED"       "wgs_PCT_EXC_CAPPED"        
#&gt; [21] "wgs_PCT_EXC_TOTAL"          "wgs_PCT_5X"                
#&gt; [23] "wgs_PCT_10X"                "wgs_PCT_20X"               
#&gt; [25] "wgs_PCT_50X"                "wgs_PCT_100X"              
#&gt; [27] "PCT_ON_BAIT"                "PCT_NEAR_BAIT"             
#&gt; [29] "MEAN_READ_LENGTH"           "BAIT_TERRITORY"            
#&gt; [31] "TARGET_TERRITORY"           "MEAN_BAIT_COVERAGE"        
#&gt; [33] "MEAN_TARGET_COVERAGE"       "PCT_TARGET_BASES_2X"       
#&gt; [35] "PCT_TARGET_BASES_20X"       "PCT_TARGET_BASES_50X"      
#&gt; [37] "PCT_TARGET_BASES_100X"      "PCT_USABLE_BASES_ON_BAIT"  
#&gt; [39] "PCT_USABLE_BASES_ON_TARGET" "FOLD_ENRICHMENT"           
#&gt; [41] "HS_LIBRARY_SIZE"            "qMean"                     
#&gt; [43] "PCT_EXC_DUPE"               "PCT_EXC_BASEQ"             
#&gt; [45] "PCT_EXC_OVERLAP"            "Average Insert Size"       </div><div class='input'><span class='fu'><a href='https://www.rdocumentation.org/packages/graphics/topics/hist'>hist</a></span>(<span class='no'>x</span>$<span class='no'>Percentage_duplication</span>)</div><div class='img'><img src='target_dna_qc-1.png' alt='' width='700' height='433' /></div><div class='input'><span class='fu'><a href='https://www.rdocumentation.org/packages/graphics/topics/hist'>hist</a></span>(<span class='no'>x</span>$<span class='no'>PCT_TARGET_BASES_20X</span>)</div><div class='img'><img src='target_dna_qc-2.png' alt='' width='700' height='433' /></div><div class='input'><span class='fu'><a href='https://www.rdocumentation.org/packages/graphics/topics/hist'>hist</a></span>(<span class='no'>x</span>$<span class='no'>PCT_NEAR_BAIT</span> + <span class='no'>x</span>$<span class='no'>PCT_ON_BAIT</span>)</div><div class='img'><img src='target_dna_qc-3.png' alt='' width='700' height='433' /></div><div class='input'>
</div></span></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
            
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>

      <footer>
      <div class="copyright">
  <p>Developed by Sean Davis.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
   </div>

  

  </body>
</html>

