<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Multivariate survival analysis • Target Osteosarcoma Data</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/united/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Multivariate survival analysis">
<meta property="og:description" content="TargetOsteoAnalysis">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-93043521-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-93043521-1');
</script>
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">Target Osteosarcoma Data</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.3.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/lcc_variants.html">Analysis of variants for LCC genes</a>
    </li>
    <li>
      <a href="../articles/multivariate_survival.html">Multivariate survival analysis</a>
    </li>
    <li>
      <a href="../articles/shatterseq.html">ShatterSeq</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/seandavi/TargetOsteoAnalysis/">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="multivariate_survival_files/htmlwidgets-1.5.1/htmlwidgets.js"></script><script src="multivariate_survival_files/jquery-1.12.4/jquery.min.js"></script><link href="multivariate_survival_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="multivariate_survival_files/datatables-binding-0.13/datatables.js"></script><link href="multivariate_survival_files/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="multivariate_survival_files/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="multivariate_survival_files/dt-core-1.10.20/js/jquery.dataTables.min.js"></script><link href="multivariate_survival_files/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet">
<script src="multivariate_survival_files/crosstalk-1.1.0.1/js/crosstalk.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Multivariate survival analysis</h1>
            <h3 class="subtitle">Application to TARGET Osteosarcoma metastatic and single sample GSEA results</h3>
                        <h4 class="author">Sean Davis<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>
</h4>
            
            <h4 class="date">2020-05-20</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/seandavi/TargetOsteoAnalysis/blob/master/vignettes/multivariate_survival.Rmd"><code>vignettes/multivariate_survival.Rmd</code></a></small>
      <div class="hidden name"><code>multivariate_survival.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      The goal of this workflow is to showcase how to use Cox regression in R to analyze a combination of continuous and categorical predictors of survival. The dataset is from the TARGET osteosarcoma project. The continuous variable(s) are derived elsewhere, but are the results of applying single sample Gene Set Enrichment Analysis (GSEA) to a group of “interesting” biological genes.
    </div>
    
<div id="learning-goals-and-objectives" class="section level2">
<h2 class="hasAnchor">
<a href="#learning-goals-and-objectives" class="anchor"></a>Learning goals and objectives</h2>
<ul>
<li>Know that Kaplan-Meier is a <em>univariate</em> statistical approach.</li>
<li>Know that Kaplan-Meier is applicable only to categorical variables.</li>
<li>Know that Cox regression can be applied to <em>univariate</em> or <em>multivariate</em> data with both categorical or continuous predictors.</li>
<li>Understand how to use <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code> to determine the statistical significance of predictors.</li>
<li>Be able to apply <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code> to determine if an additional predictor is helpful to predicting survival.</li>
<li>Be able to identify and interpret the Hazard Ratios from a Cox regression model.</li>
</ul>
</div>
<div id="setup" class="section level2">
<h2 class="hasAnchor">
<a href="#setup" class="anchor"></a>Setup</h2>
<p>We need a few R packages to get going. If these are not installed, install them first. To install the <code>TargetOsteoAnalysis</code> package requires a little extra work:</p>
<div class="sourceCode" id="cb1"><html><body><pre class="r"><span class="kw pkg">BiocManager</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span>(<span class="st">'seandavi/TargetOsteoAnalysis'</span>)</pre></body></html></div>
<div class="sourceCode" id="cb2"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">TargetOsteoAnalysis</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">survival</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">survminer</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">tidytidbits</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">dplyr</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">ggplot2</span>)</pre></body></html></div>
</div>
<div id="the-dataset" class="section level2">
<h2 class="hasAnchor">
<a href="#the-dataset" class="anchor"></a>The dataset</h2>
<p>The following code loads data from the <code>TargetOsteoAnalysis</code> package and then manipulates it to be a bit simpler for the workflow below.</p>
<div class="sourceCode" id="cb3"><html><body><pre class="r"><span class="no">ssgsea_survival</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/utils/read.table.html">read.delim</a></span>(
  <span class="fu"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span>(<span class="st">'extdata/All_ssGSEA_CTA4surv.txt.gz'</span>,
              <span class="kw">package</span><span class="kw">=</span><span class="st">'TargetOsteoAnalysis'</span>),
  <span class="kw">sep</span><span class="kw">=</span><span class="st">"\t"</span>) <span class="kw">%&gt;%</span>
  <span class="kw pkg">dplyr</span><span class="kw ns">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html">mutate</a></span>(<span class="kw">status</span><span class="kw">=</span><span class="no">Vital.Status</span><span class="kw">==</span><span class="st">'Dead'</span>,
                <span class="kw">time</span> <span class="kw">=</span> <span class="no">Overall.Survival.Time.in.Days</span>,
                <span class="kw">gender</span> <span class="kw">=</span> <span class="kw pkg">stringr</span><span class="kw ns">::</span><span class="fu"><a href="https://stringr.tidyverse.org/reference/str_trim.html">str_trim</a></span>(<span class="no">Gender</span>),
                <span class="kw">metastatic</span> <span class="kw">=</span> <span class="no">Disease.at.diagnosis</span>) <span class="kw">%&gt;%</span>
  <span class="kw pkg">dplyr</span><span class="kw ns">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html">select</a></span>(<span class="no">status</span>,<span class="no">time</span>,<span class="no">metastatic</span>,<span class="no">CTA_chrX</span>,<span class="no">CTA_all</span>,<span class="no">gender</span>)</pre></body></html></div>
<div class="sourceCode" id="cb4"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">DT</span>)
<span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html">datatable</a></span>(<span class="no">ssgsea_survival</span>)</pre></body></html></div>
<div id="htmlwidget-70a24a97de554e1fa3c3" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-70a24a97de554e1fa3c3">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91"],[false,false,false,true,true,false,false,false,true,true,false,false,false,true,false,false,true,true,true,false,false,true,true,true,true,true,true,false,false,false,true,true,false,false,false,false,true,true,false,false,true,true,true,false,true,false,false,true,true,false,false,false,false,false,false,true,false,false,false,true,true,false,true,true,false,false,false,false,false,false,false,false,false,false,false,true,false,true,true,false,false,false,true,false,true,true,false,false,false,false,false],[377,1750,2948,290,519,2216,683,1844,1637,619,1616,1583,708,627,619,5840,1906,909,788,983,2257,503,378,1023,361,388,756,922,1,2462,1515,754,4374,3282,2122,2163,2882,857,1913,3900,3862,180,74,3002,271,3378,3378,776,295,2256,3095,1217,1323,1538,1719,422,2520,2526,286,679,653,2610,1631,1061,2049,2112,2062,1747,1707,1865,1625,1870,1702,1451,1575,758,1580,1579,386,1468,1029,687,537,672,347,606,579,551,511,369,476],["Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Metastatic","Metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic","Non-metastatic"],[-1990.63,-8331.04,-8417.97,-7650.73,-8564.25,-8110.91,-5890.03,-4047.23,-7099.73,-6281.67,-5432.16,-7684.96,-4734.94,-6284.43,-3067.63,-7621.62,-7777.49,-8096.12,-7949.38,-1848.73,-7703.06,-8453.36,-8279.53,-4376.45,-1196.43,-8600.55,-7556.8,-2993.14,-8205.66,-7593.96,-6803.49,-7546.69,-2302.13,-1121.29,-3674.08,-6911.1,-8371.46,-6172.22,-4762.06,-6690.09,-4289.04,-7734.33,-8172.29,-8143.9,-8353.64,-4390.29,-7852.79,-8117.63,-8072.79,-6648.69,-8005.64,-2831.74,-7321.04,-7547.33,-6937.53,-8214.07,-7409.11,-2455.44,-8287.85,-8490.14,-8318.81,-4725.49,-6933.87,-8243.25,-8467.49,-7597.13,-1826.2,-8557.94,-6620.73,-7275.68,-4513.79,-7785.15,-7478.62,-7371.45,-3019.21,-8113.92,-2146.68,-7243.2,-8059.18,-3371.64,-6044.67,-6090.69,-7862.33,-7336.28,-3270.76,-6932.58,-4920.69,-7765.51,-7754.77,-7146.73,-6857.73],[-3241.68,-6732.44,-6966.2,-6443.38,-7195.79,-6331.02,-5353.5,-4583.55,-5976.39,-5635.22,-5500.2,-6512.96,-5122.7,-5489.21,-4078.01,-6540.84,-6487.21,-6582.95,-6601.58,-3236.72,-6713.87,-7123.21,-6583.6,-4949.77,-3451.4,-7134.52,-6420.49,-3942.67,-6504.16,-6426.55,-6092.52,-6610.44,-3527.66,-2840.08,-4447.87,-6081.79,-7007.37,-5651.13,-5111.93,-5767.3,-5004.66,-6454.18,-6904.21,-6781.59,-7017.79,-4775.8,-6665.16,-6716.31,-6768.02,-5642.25,-6791.75,-4012.22,-6555.93,-6373.43,-6105.93,-5959.06,-6860.86,-3731.45,-6750.42,-6868.54,-7038.89,-4812.8,-6131.7,-7079.57,-6688.09,-6120.85,-4005.99,-7066.26,-5852.99,-6169.71,-4817.8,-6903.3,-6258.18,-6199.04,-4117.29,-6436.05,-3727.53,-6045.57,-6765.48,-4273.85,-5763.95,-5676.09,-6557.29,-6199.25,-3286.11,-6023.15,-4762.68,-6420.81,-6509.26,-6188.15,-6028.32],["Male","Male","Male","Male","Female","Female","Male","Male","Male","Female","Female","Female","Female","Male","Male","Male","Male","Male","Male","Male","Male","Male","Male","Male","Female","Male","Male","Male","Male","Female","Male","Female","Male","Female","Male","Female","Male","Male","Male","Female","Male","Female","Female","Female","Male","Male","Male","Male","Male","Male","Female","Male","Female","Male","Female","Male","Female","Female","Female","Male","Female","Male","Female","Male","Male","Male","Female","Male","Female","Male","Female","Female","Male","Male","Male","Male","Female","Male","Female","Male","Female","Male","Female","Male","Female","Female","Female","Male","Female","Male","Female"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>status<\/th>\n      <th>time<\/th>\n      <th>metastatic<\/th>\n      <th>CTA_chrX<\/th>\n      <th>CTA_all<\/th>\n      <th>gender<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="introduction" class="section level2">
<h2 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h2>
<p>The Cox proportional-hazards model <span class="citation">(Cox 1972)</span> is semi-parametric regression model commonly used for investigating the association between the survival time of patients and one or more predictor variables. Kaplan-Meier curves and logrank tests - are examples of <em>univariate</em> analysis approaches. They can be applied in the setting of one factor under investigation, but ignore the impact of any others. Additionally, Kaplan-Meier curves and logrank tests are useful only when the predictor variable is categorical (e.g.: treatment A vs treatment B; males vs females). They don’t work easily for quantitative predictors such as gene expression, weight, or age without binning the variable first to create categories</p>
<p>Cox proportional hazards regression analysis works for both quantitative predictor variables and for categorical variables. Furthermore, the Cox regression model extends survival analysis methods to assess simultaneously the effect of several risk factors on survival time; ie., Cox regression can be <em>multivariate</em>.</p>
</div>
<div id="kaplan-meierlogrank-test-vs-cox-regression" class="section level2">
<h2 class="hasAnchor">
<a href="#kaplan-meierlogrank-test-vs-cox-regression" class="anchor"></a>Kaplan-Meier/LogRank test vs Cox Regression</h2>
<div class="sourceCode" id="cb5"><html><body><pre class="r"><span class="no">surv_met_fit</span><span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/survfit.html">survfit</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)
<span class="kw pkg">survminer</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survminer/man/ggsurvplot.html">ggsurvplot</a></span>(<span class="no">surv_met_fit</span>, <span class="kw">risk.table</span> <span class="kw">=</span> <span class="fl">TRUE</span>)</pre></body></html></div>
<div class="figure">
<img src="multivariate_survival_files/figure-html/unnamed-chunk-4-1.png" alt="Simple Kaplan-Meier survival plot with metastatic status at diagnosis." width="100%"><p class="caption">
Simple Kaplan-Meier survival plot with metastatic status at diagnosis.
</p>
</div>
<p>To perform the actual logrank testing, we apply the <code><a href="https://rdrr.io/pkg/survival/man/survdiff.html">survdiff()</a></code> function. This function tests if there is a difference between two or more survival curves using the <em>G-rho</em> family of tests <span class="citation">(Harrington and Fleming 1982)</span>.</p>
<div class="sourceCode" id="cb6"><html><body><pre class="r"><span class="no">surv_km</span>  <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/survdiff.html">survdiff</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)
<span class="no">surv_km</span>
<span class="co">#&gt; Call:</span>
<span class="co">#&gt; survdiff(formula = survival::Surv(time, status) ~ metastatic, </span>
<span class="co">#&gt;     data = ssgsea_survival)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;                            N Observed Expected (O-E)^2/E (O-E)^2/V</span>
<span class="co">#&gt; metastatic=Metastatic     22       15     5.96     13.74      16.7</span>
<span class="co">#&gt; metastatic=Non-metastatic 69       20    29.04      2.82      16.7</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  Chisq= 16.7  on 1 degrees of freedom, p= 4e-05</span></pre></body></html></div>
<p>So, the p-value for this test, also known as the <em>log-rank</em> or <em>Mantel-Haenszel</em> test, is 4.304057810^{-5}.</p>
<p>Regression approaches allow us to treat categorical and continuous variables in a common statistical framework. Here, we apply the <code><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph()</a></code> method to perform Cox regression with the categorical variable, <code>metastatic</code>, as the predictor. Note that the <strong><em>exact same</em></strong> model formula is used here, but the testing procedure is different.</p>
<div class="sourceCode" id="cb7"><html><body><pre class="r"><span class="no">surv_met</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)</pre></body></html></div>
<p>We can look at the Cox regression results using the <code><a href="https://rdrr.io/r/base/summary.html">summary()</a></code> method:</p>
<div class="sourceCode" id="cb8"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span>(<span class="no">surv_met</span>)
<span class="co">#&gt; Call:</span>
<span class="co">#&gt; survival::coxph(formula = survival::Surv(time, status) ~ metastatic, </span>
<span class="co">#&gt;     data = ssgsea_survival)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;   n= 91, number of events= 35 </span>
<span class="co">#&gt; </span>
<span class="co">#&gt;                             coef exp(coef) se(coef)      z Pr(&gt;|z|)    </span>
<span class="co">#&gt; metastaticNon-metastatic -1.3177    0.2677   0.3451 -3.818 0.000134 ***</span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;                          exp(coef) exp(-coef) lower .95 upper .95</span>
<span class="co">#&gt; metastaticNon-metastatic    0.2677      3.735    0.1361    0.5266</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Concordance= 0.664  (se = 0.041 )</span>
<span class="co">#&gt; Likelihood ratio test= 12.97  on 1 df,   p=3e-04</span>
<span class="co">#&gt; Wald test            = 14.58  on 1 df,   p=1e-04</span>
<span class="co">#&gt; Score (logrank) test = 16.73  on 1 df,   p=4e-05</span></pre></body></html></div>
<p>Digging through this result can be a bit challenging. Let’s start at the bottom. The three results, <code>Likelihood ratio test</code>, <code>Wald test</code>, and <code>Score (logrank) test</code> with <code>p=...</code> are <em>omnibus</em> tests that report the results of three different testing procedures for <strong><em>the entire model</em></strong>. In particular, note that the p-value for the <code>Score (logrank) test</code> is identical to the p-value of the <code>logrank</code> test from the <code>survdiff</code> results above.</p>
<p>The next block of rows (<code><a href="https://rdrr.io/r/base/Log.html">exp(coef)</a></code>, <code><a href="https://rdrr.io/r/base/Log.html">exp(-coef)</a></code>, <code>lower .95</code>, <code>upper .95</code>) report the hazard ratios. The <code><a href="https://rdrr.io/r/base/Log.html">exp(coef)</a></code> is the <em>Hazard Ratio</em> associated with <code>metastaticNon-metastatic</code> condition. This may seem a strange way to report things, but the assumption is that the “baseline” is the <code>metasticMetastatic</code> condition; thus, the hazard ratio reported here, 0.2677, suggests that “Non-metastatic” is <em>protective</em>. The “lower” and “upper” confidence intervals (<code>lower .95</code>, <code>upper .95</code>) do not intersect a hazard ratio of 1, so we would expect the effect to be significant.</p>
<p>Moving up to the next block of rows, we can find the p-value for the specific variable (<code>Pr(&gt;|z|)</code>). The p-value reported here is <em>identical</em> to the p-value for the Wald Test, 1.343327410^{-4}.</p>
</div>
<div id="looking-at-cta-genes" class="section level2">
<h2 class="hasAnchor">
<a href="#looking-at-cta-genes" class="anchor"></a>Looking at CTA genes</h2>
<p>We can apply the same Cox regression procedure for the continuous variable, <code>CTA_all</code>, for example.</p>
<div class="sourceCode" id="cb9"><html><body><pre class="r"><span class="no">surv_cta_all</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">CTA_chrX</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)
<span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span>(<span class="no">surv_cta_all</span>)
<span class="co">#&gt; Call:</span>
<span class="co">#&gt; survival::coxph(formula = survival::Surv(time, status) ~ CTA_chrX, </span>
<span class="co">#&gt;     data = ssgsea_survival)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;   n= 91, number of events= 35 </span>
<span class="co">#&gt; </span>
<span class="co">#&gt;                coef  exp(coef)   se(coef)      z Pr(&gt;|z|)   </span>
<span class="co">#&gt; CTA_chrX -0.0003237  0.9996764  0.0001206 -2.683  0.00729 **</span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;          exp(coef) exp(-coef) lower .95 upper .95</span>
<span class="co">#&gt; CTA_chrX    0.9997          1    0.9994    0.9999</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Concordance= 0.679  (se = 0.05 )</span>
<span class="co">#&gt; Likelihood ratio test= 9.9  on 1 df,   p=0.002</span>
<span class="co">#&gt; Wald test            = 7.2  on 1 df,   p=0.007</span>
<span class="co">#&gt; Score (logrank) test = 8.01  on 1 df,   p=0.005</span></pre></body></html></div>
<p>Again, looking at the <em>omnibus</em> tests, all are significant for the entire model. The Hazard Ratio estimate is 0.9997 which is interpreted to mean that higher ssGSEA scores for the CTA genes is protective.</p>
<p>Now, how can we include metastasis <em>and</em> the ssGSEA results in the same model? Programmatically, we can just include a <code><a href="https://rdrr.io/r/base/Arithmetic.html">+</a></code> in the model formula. That looks like:</p>
<div class="sourceCode" id="cb10"><html><body><pre class="r"><span class="no">surv_full</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span> + <span class="no">CTA_all</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)</pre></body></html></div>
<p>Does adding <code>CTA_all</code> to the model make a significance difference? The way to address the question of whether a model that includes more covariates is “better” than one with fewer is to formally test. In particular, we need to have the model without the CTA_all results and compare it to the same model but with the CTA_all results included.</p>
<p>The model with only metastasis looks like:</p>
<div class="sourceCode" id="cb11"><html><body><pre class="r"><span class="no">surv_met_only</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)</pre></body></html></div>
<p>We use the <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code> method to compare models:</p>
<div class="sourceCode" id="cb12"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span>(<span class="no">surv_met_only</span>,<span class="no">surv_full</span>)
<span class="co">#&gt; Analysis of Deviance Table</span>
<span class="co">#&gt;  Cox model: response is  survival::Surv(time, status)</span>
<span class="co">#&gt;  Model 1: ~ metastatic</span>
<span class="co">#&gt;  Model 2: ~ metastatic + CTA_all</span>
<span class="co">#&gt;    loglik  Chisq Df P(&gt;|Chi|)   </span>
<span class="co">#&gt; 1 -133.51                       </span>
<span class="co">#&gt; 2 -128.78 9.4759  1  0.002082 **</span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></pre></body></html></div>
<p>The results are comparing model <code>1</code> that includes only metastasis as a covariate vs model <code>2</code> that includes both metastasis <em>and</em> CTA_all. The equivalence of the two models is easily “rejected” since the p-value (<code>P(&gt;|Chi|)</code>) is small.</p>
<p>R has a convenient means to examine a single model using <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code>. If we have the full model only, for example, calling <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code> on that model will return the results of each “submodel” with variables added in the order of the model formula. You’ll note above that I specified <code>metastatic + CTA_all</code>, so the base model will include no covariates, model 1 will include metastatic status, and model 2 will include both covariates.</p>
<div class="sourceCode" id="cb13"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span>(<span class="no">surv_full</span>)
<span class="co">#&gt; Analysis of Deviance Table</span>
<span class="co">#&gt;  Cox model: response is survival::Surv(time, status)</span>
<span class="co">#&gt; Terms added sequentially (first to last)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;             loglik   Chisq Df Pr(&gt;|Chi|)    </span>
<span class="co">#&gt; NULL       -140.00                          </span>
<span class="co">#&gt; metastatic -133.51 12.9740  1  0.0003158 ***</span>
<span class="co">#&gt; CTA_all    -128.78  9.4759  1  0.0020819 ** </span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></pre></body></html></div>
<p>We can see that adding <code>metastatic</code> is signficant. Given that <code>metastatic</code> is in the model, adding <code>CTA_all</code> is also significant.</p>
<p>What about CTA_chrX as a covariate?</p>
<div class="sourceCode" id="cb14"><html><body><pre class="r"><span class="no">surv_full_X</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span> + <span class="no">CTA_chrX</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)
<span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span>(<span class="no">surv_full_X</span>)
<span class="co">#&gt; Analysis of Deviance Table</span>
<span class="co">#&gt;  Cox model: response is survival::Surv(time, status)</span>
<span class="co">#&gt; Terms added sequentially (first to last)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;             loglik  Chisq Df Pr(&gt;|Chi|)    </span>
<span class="co">#&gt; NULL       -140.00                         </span>
<span class="co">#&gt; metastatic -133.51 12.974  1  0.0003158 ***</span>
<span class="co">#&gt; CTA_chrX   -128.01 11.003  1  0.0009099 ***</span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></pre></body></html></div>
<p>So, it looks like <code>CTA_chrX</code> results also add value to the model. How about including everything?</p>
<p>Including both “CTA” result sets looks like:</p>
<div class="sourceCode" id="cb15"><html><body><pre class="r"><span class="no">surv_all</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span> + <span class="no">CTA_chrX</span> + <span class="no">CTA_all</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)
<span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span>(<span class="no">surv_all</span>)
<span class="co">#&gt; Analysis of Deviance Table</span>
<span class="co">#&gt;  Cox model: response is survival::Surv(time, status)</span>
<span class="co">#&gt; Terms added sequentially (first to last)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;             loglik   Chisq Df Pr(&gt;|Chi|)    </span>
<span class="co">#&gt; NULL       -140.00                          </span>
<span class="co">#&gt; metastatic -133.51 12.9740  1  0.0003158 ***</span>
<span class="co">#&gt; CTA_chrX   -128.01 11.0026  1  0.0009099 ***</span>
<span class="co">#&gt; CTA_all    -127.66  0.7079  1  0.4001532    </span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></pre></body></html></div>
<p>And, for completeness, let’s switch the ordering of the CTA sets?</p>
<div class="sourceCode" id="cb16"><html><body><pre class="r"><span class="no">surv_all</span> <span class="kw">=</span> <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">metastatic</span> + <span class="no">CTA_all</span> + <span class="no">CTA_chrX</span>,
  <span class="kw">data</span><span class="kw">=</span><span class="no">ssgsea_survival</span>)
<span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span>(<span class="no">surv_all</span>)
<span class="co">#&gt; Analysis of Deviance Table</span>
<span class="co">#&gt;  Cox model: response is survival::Surv(time, status)</span>
<span class="co">#&gt; Terms added sequentially (first to last)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;             loglik   Chisq Df Pr(&gt;|Chi|)    </span>
<span class="co">#&gt; NULL       -140.00                          </span>
<span class="co">#&gt; metastatic -133.51 12.9740  1  0.0003158 ***</span>
<span class="co">#&gt; CTA_all    -128.78  9.4759  1  0.0020819 ** </span>
<span class="co">#&gt; CTA_chrX   -127.66  2.2346  1  0.1349540    </span>
<span class="co">#&gt; ---</span>
<span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></pre></body></html></div>
<p>Why does adding another CTA set not help? Looking at the relationship between the two CTA result sets is helpful.</p>
<div class="sourceCode" id="cb17"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/with.html">with</a></span>(<span class="no">ssgsea_survival</span>, <span class="fu"><a href="https://rdrr.io/r/base/plot.html">plot</a></span>(<span class="no">CTA_chrX</span>, <span class="no">CTA_all</span>))</pre></body></html></div>
<div class="figure">
<img src="multivariate_survival_files/figure-html/ssgsea-1.png" alt="Scatterplot of single sample GSEA scores showing very high correlation between the two" width="100%"><p class="caption">
Scatterplot of single sample GSEA scores showing very high correlation between the two
</p>
</div>
<p>With such high correlation between the two sets (see Figure @ref(fig:ssgsea)), we can understand how including one in the model makes the other irrelevant.</p>
</div>
<div id="splitting-into-two-groups-first" class="section level2">
<h2 class="hasAnchor">
<a href="#splitting-into-two-groups-first" class="anchor"></a>Splitting into two groups first</h2>
<p>If we had split the data into “met” and “no-met” groups and then performed the Cox regression, we get the following results:</p>
<div class="sourceCode" id="cb18"><html><body><pre class="r"><span class="no">met_choices</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html">unique</a></span>(<span class="no">ssgsea_survival</span>$<span class="no">metastatic</span>)
<span class="no">res</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span>(<span class="no">met_choices</span>, <span class="kw">function</span>(<span class="no">met</span>) {
  <span class="no">df</span> <span class="kw">=</span> <span class="no">ssgsea_survival</span> <span class="kw">%&gt;%</span> <span class="kw pkg">dplyr</span><span class="kw ns">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html">filter</a></span>(<span class="no">metastatic</span><span class="kw">==</span><span class="no">met</span>)
  <span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(<span class="kw pkg">survival</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">time</span>,<span class="no">status</span>) ~ <span class="no">CTA_chrX</span>, <span class="kw">data</span><span class="kw">=</span><span class="no">df</span>)
})
<span class="fu"><a href="https://rdrr.io/r/base/names.html">names</a></span>(<span class="no">res</span>) <span class="kw">=</span> <span class="no">met_choices</span>
<span class="kw pkg">knitr</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(
  <span class="kw">label</span> <span class="kw">=</span> <span class="st">'metsep'</span>,
  <span class="kw pkg">dplyr</span><span class="kw ns">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/bind.html">bind_rows</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span>(<span class="no">res</span>,<span class="kw pkg">broom</span><span class="kw ns">::</span><span class="no"><a href="https://rdrr.io/pkg/broom/man/reexports.html">tidy</a></span>),<span class="kw">.id</span><span class="kw">=</span><span class="st">'met_status'</span>),
  <span class="kw">caption</span> <span class="kw">=</span> <span class="st">"Cox proportional hazards models to determine the effect of CTA genes on chromosome X (from ssGSEA analysis), applied separately to metastatic and non-metastatic patients. Note that the p-values are not significant when patients are treated separately."</span>
)</pre></body></html></div>
<table class="table">
<caption>Cox proportional hazards models to determine the effect of CTA genes on chromosome X (from ssGSEA analysis), applied separately to metastatic and non-metastatic patients. Note that the p-values are not significant when patients are treated separately.</caption>
<thead><tr class="header">
<th align="left">met_status</th>
<th align="left">term</th>
<th align="right">estimate</th>
<th align="right">std.error</th>
<th align="right">statistic</th>
<th align="right">p.value</th>
<th align="right">conf.low</th>
<th align="right">conf.high</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">Non-metastatic</td>
<td align="left">CTA_chrX</td>
<td align="right">-0.0002146</td>
<td align="right">0.0001384</td>
<td align="right">-1.550350</td>
<td align="right">0.1210575</td>
<td align="right">-0.0004858</td>
<td align="right">5.67e-05</td>
</tr>
<tr class="even">
<td align="left">Metastatic</td>
<td align="left">CTA_chrX</td>
<td align="right">-0.0005132</td>
<td align="right">0.0002662</td>
<td align="right">-1.927684</td>
<td align="right">0.0538944</td>
<td align="right">-0.0010349</td>
<td align="right">8.60e-06</td>
</tr>
</tbody>
</table>
<p>Checking the p-values for the CTA_chrX covariate in Table @ref(tab:metsep) in <em>each group separately</em>, we see that they are trending in the right direction, but are not significant.</p>
</div>
<div id="summary" class="section level2">
<h2 class="hasAnchor">
<a href="#summary" class="anchor"></a>Summary</h2>
<ul>
<li>Kaplan-Meier and logrank tests have limited application to a single categorical variable</li>
<li>Cox regression is applicable for both continuous and categorical and can be applied to multiple variables as once.</li>
</ul>
<p>When working with Cox regression:</p>
<ul>
<li>To test the significance of a variable, apply <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code> to the model.</li>
<li>To see the hazard ratios associated with a condition, use <code><a href="https://rdrr.io/r/base/summary.html">summary()</a></code> and look at the <code><a href="https://rdrr.io/r/base/Log.html">exp(coef)</a></code>.</li>
<li>To determine if adding a variable improves a model, compare the model to a simpler model without the extra variable using <code><a href="https://rdrr.io/r/stats/anova.html">anova()</a></code>.</li>
</ul>
</div>
<div id="references" class="section level2 unnumbered">
<h2 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h2>
<div id="refs" class="references">
<div id="ref-Cox1972-cj">
<p>Cox, D R. 1972. “Regression Models and Life-Tables.” <em>Journal of the Royal Statistical Society. Series B, Statistical Methodology</em> 34 (2):187–202. <a href="https://doi.org/10.1111/j.2517-6161.1972.tb00899.x" class="uri">https://doi.org/10.1111/j.2517-6161.1972.tb00899.x</a>.</p>
</div>
<div id="ref-Harrington1982-ke">
<p>Harrington, David P, and Thomas R Fleming. 1982. “A Class of Rank Test Procedures for Censored Survival Data.” <em>Biometrika</em> 69 (3). [Oxford University Press, Biometrika Trust]:553–66. <a href="https://doi.org/10.2307/2335991" class="uri">https://doi.org/10.2307/2335991</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr>
<ol>
<li id="fn1"><p><a href="mailto:seandavi@gmail.com">seandavi@gmail.com</a><a href="#fnref1" class="footnote-back">↩</a></p></li>
</ol>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Sean Davis.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
